### DISCUSSION TITLE
Web server's domain setup question.

### DISCUSSION THREAD

**Discussion by ziozzang** [lablup/backend.ai#2515] [Q&A]
Since the last event of hands-on session, I've been testing the installation on my local machine and want to proxy requests accessing port 8090 through HTTPS communication.

It seems that I need to change the settings accordingly, but I can't find the relevant parts in the configuration. The most likely area appears to be the webserver settings, but I'm not sure if there are any specific settings there either.
- checked: https://github.com/lablup/backend.ai/blob/main/configs/webserver/sample.conf

When I request a web page, config.toml is transmitted. In its contents, I can see that the endpoint address is exposed as an internal address. It seems that requests are made to addresses under func/ based on this address.

[general]
apiEndpoint = "http://10.2.3.4:8090"

Can you provide information about the relevant settings for this?

**Comment by hoyajigi** [comment]
Thanks for your interest in Backend.AI

The relevant setting in webserver.conf is :
```
# If you need an SSL-enabled server, fill these configurations.
ssl_enabled = true
ssl_cert = <path to your cert file>
ssl_privkey = <path to your private key file>
```

You can write as:
```
ssl_enabled = true
ssl_cert = "/home/ubuntu/certchain.pem"
ssl_privkey = "/home/ubuntu/private.key"
```

Often the private key is stored in the same file as the certificate; in this case, you can use the same file for both.
```
ssl_enabled = true
ssl_cert = "/home/ubuntu/combined.pem"
ssl_privkey = "/home/ubuntu/combined.pem"
```

If you have a favorite reverse proxy server like Nginx or HAProxy, you can also use that to terminate SSL and forward requests to the webserver instead of enabling SSL in the webserver itself.

If you have any questions, please feel free to ask.
